nltk/tokenize/stanford_segmenter.py

```diff
@@ -70,9 +70,8 @@ def __init__(self,
                 searchpath=(), url=_stanford_url,
```

Comment:
I have removed the redundant parts and finished the tests locally. In addition, I think since the stanford-segmenter-2016-10-31 is released 7 months ago, it's acceptable to move forward.